Source/WebCore/html/ColorInputType.cpp

```diff
@@ -348,11 +347,11 @@ HTMLElement* ColorInputType::shadowColorSwatch() const
     if (!shadow)
         return nullptr;
 
-    RefPtr wrapper = childrenOfType<HTMLDivElement>(*shadow).first();
+    RefPtr wrapper = shadow->firstChild();
     if (!wrapper)
         return nullptr;
```

Comment:
Note that while I removed some of the code, I have generally preserved the idea that we have to deal with these elements potentially not existing. Strictly speaking even that shouldn't be needed, but I'm not sure how comfortable we are with the shadow boundary being inpenetrable these days.